network/mikrotik/snmp/mode/memory.pm

```diff
@@ -0,0 +1,105 @@
+#
+# Copyright 2018 Centreon (http://www.centreon.com/)
+#
+# Centreon is a full-fledged industry-strength solution that meets
+# the needs in IT infrastructure and application monitoring for
+# service performance.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+package network::mikrotik::snmp::mode::memory;
```

Comment:
please try with snmp standard mode

Reply:
Sort of the same situation as the cpu, will do the same test

On hold for testing

Reply:
Stress test complete, the standard paths for memory do not return any values so we have to keep this file

Reply:
Stress test complete, paths in standard_snmp do not return any values on our mikrotik appliances, this file needs to be kept